_pages/support.md

```diff
@@ -392,4 +411,60 @@ SAML requests from browser consoles are URI encoded, base-64-encoded, and deflat
 </div>
 </div>
 
+<h4 class="usa-accordion__heading">
+<button class="usa-accordion__button" aria-controls="samlsignaturetroubleshooting">
+SAML Signature Troubleshooting
+</button>
+</h4>
+<div id="samlsignaturetroubleshooting" class="usa-accordion__container">
+<div class="usa-accordion__content" markdown="1"  aria-expanded="true"> 
+Login.gov uses the cryptographic signatures of authentication requests to determine which public certificate to use when encrypting data in the SAML response. If the signature is not present, or cannot be validated successfully, you will encounter problems when you rotate your application’s key pair.
+
+<b> Check signature is present </b>
+
+If you are not sure whether your application is currently signing authentication requests, the easiest way to check is through the network tab in your browser's developer tools. Look for the URL generated when your app sends the user to Login.gov's sign-in page.
+
+The signature could be sent in one of two ways:
+<div>
+<ol class="usa-list">
```

Comment:
**[suggestion]** i am not sure where we're standing on correctly formatting html elements, since they're not being formatted in so much of this page, but in places that we do format we should do it correctly, eg:

```html
<div>
    <ol class="usa-list">
      <li>As a parameter within the URL,</li>
      <li>or as a field within the authentication request’s SAML data.</li>
   </ol>
</div>
```